templates/games/card-game.html

```diff
@@ -1,28 +1,159 @@
+                <div class="card card-back card-hard" id="card-hard-4"></div>
+                <div class="card card-back card-hard" id="card-hard-5"></div>
+                <div class="card card-back card-hard" id="card-hard-6"></div>
+                <div class="card card-back card-hard" id="card-hard-7"></div>
+                <div class="card card-back card-hard" id="card-hard-8"></div>
+                <div class="card card-back card-hard" id="card-hard-9"></div>
+                <div class="card card-back card-hard" id="card-hard-10"></div>
+                <div class="card card-back card-hard" id="card-hard-11"></div>
+                <div class="card card-back card-hard" id="card-hard-12"></div>
+                <div class="card card-back card-hard" id="card-hard-13"></div>
+                <div class="card card-back card-hard" id="card-hard-14"></div>
+                <div class="card card-back card-hard" id="card-hard-15"></div>
+                <div class="card card-back card-hard" id="card-hard-16"></div>
+                <div class="card card-back card-hard" id="card-hard-17"></div>
+                <div class="card card-back card-hard" id="card-hard-18"></div>
+                <div class="card card-back card-hard" id="card-hard-19"></div>
+                <div class="card card-back card-hard" id="card-hard-20"></div>
+                <div class="card card-back card-hard" id="card-hard-21"></div>
+                <div class="card card-back card-hard" id="card-hard-22"></div>
+                <div class="card card-back card-hard" id="card-hard-23"></div>
+            </div>
+        </div>
+    </div>
+</div>
 
+<div class="container">
+    <div class="row">
+        <div class="col-sm-2 offset-sm-3 col-md-2 col-lg-2 text-center">
+            <button type="button" class="btn easy level-selector" id="easy">easy</button>
+        </div>
+        <div class="col-sm-3 col-md-3 col-lg-2 text-center">
+            <button type="button" class="btn medium level-selector" id="medium">medium</button>
+        </div>
+        <div class="col-sm-1 col-md-1 col-lg-1 text-center">
+            <button type="button" class="btn hard level-selector" id="hard">hard</button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-2 offset-sm-3 col-md-2 col-lg-2 text-center">
+            <div class="movescounter" id="movescounter">moves: 0</div>
+        </div>
+        <div class="col-sm-3 col-md-3 col-lg-2 text-center">
+            <div id="score">0 / 10 </div>
+        </div>
+        <div class="col-sm-1 col-md-1 col-lg-1 text-center">
+            <button type="button" class="btn reset" id="reset">reset</button>
+        </div>
+    </div>
+</div>
 
+<div class="modal" id="winModal">
+    <div class="congrats">
+        <h3>congratulations!!!</h3>
+    </div>
+    <div class="modal-content">
+        <p>you have found all the cards!, Would you like to play again?</p>
```

Comment:
And here, starting with a capital Y. Namely: **Y**ou have found....

Reply:
This was also was made all in uppercase on the live site